### PR TITLE
Added notes for first-time users

### PR DESCRIPTION
**Pull request by cbeach47**
Added section 'First Time Running uPortal via uPortal-start'

##### Checklist

-   [x ] commit message follows [commit guidelines][]
-   [x ] documentation is changed or added

##### Description of change

First time installing/running uPortal5 / uPortal-start and gathered some learning points into this 'first time users' section.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,13 @@ You can stop the Tomcat server using with this command:
 ```console
     $ ./gradlew tomcatStop
 ```
+### First Time Running uPortal via uPortal-start
+Assuming all the defaults were left:
+* The URL to access uPortal is:  http://localhost:8080/uPortal/
+* Using the example credentials, you can bypass CAS when testing locally.  For example: 	http://localhost:8080/uPortal/Login?userName=admin&password=admin
```
Comment (ChristianMurphy) [inline README.md]: Definitely helpful :+1: 

The full list of default logins I'm aware of is:
* admin <http://localhost:8080/uPortal/Login?userName=admin&password=admin>
* faculty <http://localhost:8080/uPortal/Login?userName=faculty&password=faculty>
* staff <http://localhost:8080/uPortal/Login?userName=staff&password=staff>
* student <http://localhost:8080/uPortal/Login?userName=student&password=student>
* guest <http://localhost:8080/uPortal/render.userLayoutRootNode.uP>

File: README.md [hunk 2]
```diff
@@ -204,6 +204,13 @@ You can stop the Tomcat server using with this command:
 ```console
     $ ./gradlew tomcatStop
 ```
+### First Time Running uPortal via uPortal-start
+Assuming all the defaults were left:
+* The URL to access uPortal is:  http://localhost:8080/uPortal/
```
Comment (ChristianMurphy) [inline README.md]: :art: please add `<>` around literal urls. E.G. `<http://localhost:8080/uPortal/>`
:question: Would/could it be helpful if uPortal automatically opened a browser to this url?

```gradle
task openUrlInBrowser << {
   java.awt.Desktop.desktop.browse 'http://localhost:8080/uPortal/'.toURI()
}
```

File: README.md [hunk 3]
```diff
@@ -204,6 +204,13 @@ You can stop the Tomcat server using with this command:
...
+* Using the example credentials, you can bypass CAS when testing locally.  For example: 	http://localhost:8080/uPortal/Login?userName=admin&password=admin
+* The default tomcat install is:  <<uPortal-start repo directory>>/.gradle/tomcat
+* The logs to watch for issues are located in:  <<uPortal-start repo directory>>/.gradle/tomcat/logs
```
Comment (ChristianMurphy) [inline README.md]: :art: uPortal has been using *italics* as style for folders and paths, uPortal has traditionally listed all paths as relative to the root project directory be default. However in this case *.gradle/* can occur in serveral different places to giving some additional context helps. 

Maybe something like:

* The default tomcat install is: *<\<uPortal-start repo directory\>>/.gradle/tomcat*
* The logs to watch for issues are located in:  *<\<uPortal-start repo directory\>>/.gradle/tomcat/logs*

```md
* The default tomcat install is: *<\<uPortal-start repo directory\>>/.gradle/tomcat*
* The logs to watch for issues are located in:  *<\<uPortal-start repo directory\>>/.gradle/tomcat/logs*
```

or

* The default tomcat install is: *uPortal-start/.gradle/tomcat*
* The logs to watch for issues are located in:  *uPortal-start/.gradle/tomcat/logs*

```md
* The default tomcat install is: *uPortal-start/.gradle/tomcat*
* The logs to watch for issues are located in:  *uPortal-start/.gradle/tomcat/logs*
```

**Comment by cbeach47** [comment]
Added some modifications as per @ChristianMurphy

**Pull request merged**